blackmarket/UI/Views/HomeView.swift

```diff
@@ -7,11 +7,35 @@
 
 import SwiftUI
 
-//TODO: Create this view
-
 struct HomeView: View {
+    
+    @State var searchText: String = ""
+
     var body: some View {
-        Text("HomeView")
+        ScrollView {
+            VStack {
+                SearchBar(text: $searchText)
+                Image("banner1")
+                PaymentMethodsView()
+                Image("banner2")
+            }
+        }
+        .padding(.top, UI.HomeView.ScrollView.topPadding)
+        .background(Color.homeBackground)
+        .padding(.bottom, UI.HomeView.ScrollView.bottomPadding)
+        .padding(.top, UI.HomeView.topPadding)
```

Comment:
pls group the related modifiers for better readability. ALso, why is the top padding being set twice?